tests/plugins/easyimage/uploadwidget.js

```diff
@@ -438,6 +430,30 @@
 			} );
 
 			wait();
+		},
+
+		'test cloudservices URL can be customized': function( editor ) {
```

Comment:
Not only URL can be customised, but also loader type and additional request parameters. It'd be nice to have also tests for these.